modules/configuring-a-recommended-accelerator-tag.adoc

```diff
@@ -0,0 +1,38 @@
+:_module-type: PROCEDURE
+
+[id="configuring-a-recommended-accelerator-tag_{context}"]
+= Configuring a recommended accelerator tag
+
+[role='_abstract']
+To help you indicate the most suitable and appropriate accelerators to your data scientists, you can configure a recommended tag to appear on the dashboard. You must apply the annotation `opendatahub.io/recommended-accelerators` to an ImageStream. 
+
+.Prerequisites
+* You have logged in to {openshift-platform}.
+* You have the `cluster-admin` role in {openshift-platform}.
+* You have configured an ImageStream that contains your notebook image settings.  
+
+.Procedure
+. In the {openshift-platform} web console, in the *Administrator* perspective, click *Builds* -> *ImageStreams*.
+. Click the ImageStream that you want to a recommended accelerator tag to. 
++
+A details page opens.
+. Click the *YAML* tab.
++ 
+A page with an embedded YAML editor opens.
```

Comment:
Where is the YAML code coming from here? Is it a specific chunk of code, or are we just trying to say 'you need to enter YAML code in this editor'?

Reply:
If they've manually written some YAML code elsewhere, such as in a text editor, they can copy the YAML from the text editor and then paste it in to the embedded YAML editor. If they don't have any pre-prepared YAML, they will need to enter the code manually in the embedded editor.